.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,13 +1,13 @@
 name: Bug Report
-description: 'Support Requests will be closed immediately, if you are not 100% certain this is a bug please go to our Reddit, Discord, Forums, or IRC first. Exceptions do not mean you found a bug!'
+description: 'Support Requests will be closed immediately, if you are not 100% certain this is a bug please go to our Reddit, Discord, Forums, or IRC first. Exceptions do not mean you found a bug! Sonarr v2 is end of life and not supported.'
 labels: ['needs-triage']
 body:
 - type: checkboxes
   attributes:
     label: Is there an existing issue for this?
-    description: Please search to see if an issue already exists for the bug you encountered.
+    description: Please search to see if an open or closed issue already exists for the bug you encountered. If a bug exists and is closed note that it may only be fixed in an unstable branch.
```

Comment:
The second sentence is both awkward to read and needlessly complicated.

> If a bug exists and it is closed as complete it may not yet be in a stable release.

Reply:
Wordings suggestions welcomed.

Intent is for users to understand that fixes may only be in dev.

Reply:
I left a suggestion in my comment... `If a bug exists and it is closed as complete it may not yet be in a stable release.`